modules/mobile/app/controllers/mobile/v0/profile_base_controller.rb

```diff
@@ -10,7 +10,7 @@ class ProfileBaseController < ApplicationController
       include Vet360::Writeable
       include Mobile::Concerns::SSOLogging
 
-      before_action { authorize :vet360, :access? }
+      before_action { authorize :vet360, :profile_access? }
```

Comment:
Vet360_ID is not required for ContactInformationV2 service. The ICN will be used when the vet360_id is null.